### DISCUSSION TITLE
How to splice the lane lines generated by prediction?

### DISCUSSION THREAD

**Discussion by zhan7236** [OpenStitching/stitching#173] [Q&A]
Thanks for reply, please. Because the lane lines in the two pictures are not completely coincident, I want to complete the splicing at the seam. If splice directly, an error will be reported. 

`stitching.stitching_error.StitchingError: No match exceeds the given confidence threshold.
                                 Do your images have enough overlap and common
                                 features? If yes, you might want to lower the
                                 'confidence_threshold' or try another
                                 'detector'.`

Even if I adjust confidence_threshold  or try another detector, it still doesn't work. The pictures that need to be spliced ​​are as follows:

![pred1](https://github.com/OpenStitching/stitching/assets/76658920/cd5d0d63-4e3e-493a-871f-4e516cdae425)
![pred2](https://github.com/OpenStitching/stitching/assets/76658920/3429b7c5-041f-4374-a026-0881f6d7da58)

**Comment by lukasalexanderweber** [comment]
these images are not suited for stitching

**Reply by zhan7236** [reply]
thanks for reply, I try to lower confidence_threshold to see if it can match the images, but it doesn't work. Could you please give me some advise to solve the problem, thanks again!

**Reply by zhan7236** [reply]
Or can I modify the number or quality of matching feature points under your framework to complete the splicing?

**Reply by lukasalexanderweber** [reply]
The problem is that there are no distinguishable features in your images.

**Reply by zhan7236** [reply]
thanks a lot, maybe that's why I change the threshold lowest but it doesn't work.
